views/account/account.handlebars

```diff
@@ -145,7 +145,10 @@
 				<div class="setting-card">
 					<h2 class="header">{{ locale.account.settings.settingCards.discord }}</h2>
 						{{#if discordUser}}
-					<p>{{ locale.account.settings.settingCards.connectedToDiscord }} {{ discordUser.username }}{{#ifCond discordUser.discriminator '#0'}}{{else}}#{{ discordUser.discriminator }}{{/ifCond}}</p>
+					<p>{{ locale.account.settings.settingCards.connectedToDiscord }} {{ discordUser.username }}
+					{{#if discordUser.discriminator}}
+					&nbsp;#{{ discordUser.discriminator }}
```

Comment:
Why exactly is there an extra space here in the form of `&nbsp;`? This now creates _2_ spaces between the username and the discriminator when it is shown

Reply:
that mustve been an accident, will remove

Reply:
The `&nbsp;` is still there btw